promscale/installation/upgrade.md

```diff
@@ -0,0 +1,70 @@
+# Upgrade Promscale
+Promscale consists of the Promscale Connector, and the Promscale PostgreSQL
+extension. When you upgrade your Promscale installation, you need to check both
+components. This section contains important information about upgrading.
+
+The process for updating your Promscale installation is:
+
+1.  Check that the most recent required version of the Promscale extension is
+    available.
+2.  Stop all Promscale Connector instances connected to the database.
+3.  Perform the upgrade.
+4.  Start the new version of the Promscale Connector, and wait for it to begin
+    ingesting data.
+5.  Start all other Promscale Connector instances.
+
+## Upgrade to Promscale&nbsp;0.11.0
```

Comment:
How/when do you decide when to use `&nbsp;`?

Reply:
It seems as though using `&nbsp;` in a header kills whatever follows the space.

Reply:
It should be a non-breaking space between the product name and version number, but if it's breaking in the header we'll need to make it a regular space, I guess.